### DISCUSSION TITLE
Exception on brand new project

### DISCUSSION THREAD

**Discussion by serialexperiment** [vercel/next.js#45640] [Help]
Hi everyone,

My background isn't in js or web development, so this may just be an issue of me not fully understanding the tooling or having something simple misconfigured.

When I create a new nextjs project and try to run it (without having changed anything), I get an "ReferenceError: exports is not defined" out of runtime.js on line 5 
`Object.defineProperty(exports, "__esModule", { value: true });`

I saw that there was an issue for this a while back and it was fixed, but I am on a version released after the fix.

Here is what I exactly what I did:

1. npx create-next-app@latest nextjs-test-app
2. cd nextjs-test-app
3. npm run dev
4. open browser, go to localhost:3000
5. blank page, exception

I've tried creating the project both with and without typescript, I've tried using old versions of nextjs and create-next-app, I've tried following various official and unofficial tutorials on setting up nextjs, and at this point I'm at a total loss.

Any help is greatly appreciated!

**Comment by ciruz** [comment]
Hey,

which node.js version have you installed?
Since you are installing the latest next.js you need at least node.js ``14.6.0``.

**Comment by ryfan** [comment]
first, check node.js version~
be sure using node.js version 14+. (lts version is very recommended)

**Comment by serialexperiment** [comment]
At first I installed node 19.6.0 current, but I uninstalled that and reinstalled 18.14.0 LTS after having these issues.  I just tried doing everything the exact same but on my work PC and _only_ installing node 18.14.0 LTS, and it works fine.  On the PC where I'm having these issues, I am completely deleting the project folder and node_modules folder between tests, but maybe the first node install cached some packages somewhere?

**Comment by serialexperiment** [comment]
Update:

I figured it out.  As it would turn out, [this](https://addons.mozilla.org/en-US/firefox/addon/anti-adblock-blocker/) Firefox extension was causing the issue.  Seeing as how it doesn't look like its been updated in 3 years, I will probably just completely uninstall it.

Thank you to everyone who gave advice!
